iothub/device/tests/IotHubModuleClientTests.cs

```diff
@@ -2,30 +2,47 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Collections.Generic;
+using System.Net.Sockets;
+using System.Net.WebSockets;
+using System.Security.Cryptography.X509Certificates;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
+using Newtonsoft.Json;
 
 namespace Microsoft.Azure.Devices.Client.Test
 {
     [TestClass]
     [TestCategory("Unit")]
     public class IotHubModuleClientTests
     {
+        private const string DeviceId = "module-twin-test";
+        private const string ModuleId = "mongo-server";
         private const string ConnectionStringWithModuleId = "GatewayHostName=edge.iot.microsoft.com;HostName=acme.azure-devices.net;DeviceId=module-twin-test;ModuleId=mongo-server;SharedAccessKey=dGVzdFN0cmluZzQ=";
         private const string ConnectionStringWithoutModuleId = "GatewayHostName=edge.iot.microsoft.com;HostName=acme.azure-devices.net;DeviceId=module-twin-test;SharedAccessKey=dGVzdFN0cmluZzQ=";
         private const string FakeConnectionString = "HostName=acme.azure-devices.net;SharedAccessKeyName=AllAccessKey;DeviceId=dumpy;ModuleId=dummyModuleId;SharedAccessKey=dGVzdFN0cmluZzE=";
+        private const string FakeHostName = "acme.azure-devices.net";
+#pragma warning disable SYSLIB0026 // Type or member is obsolete
+        private static readonly X509Certificate2 s_cert = new();
+#pragma warning restore SYSLIB0026 // Type or member is obsolete
+        private static readonly X509Certificate2Collection s_certs = new();
 
         public const string NoModuleTwinJson = "{ \"maxConnections\": 10 }";
 
+        private DirectMethodResponse _directMethodResponseWithEmptyByteArrayPayload = new(200)
+        {
+            Payload = new byte[0],
+        };
+
         [TestMethod]
-        [ExpectedException(typeof(ArgumentNullException))]
         public async Task ModuleClient_CreateFromConnectionString_NullConnectionStringThrows()
         {
-            await using var moduleClient = new IotHubModuleClient(null);
+            Func<Task> act = async () => { await using var moduleClient = new IotHubModuleClient(null); };
```

Comment:
If you just have one statement, I think you can avoid the braces.

```suggestion
            Func<Task> act = async () => await using var moduleClient = new IotHubModuleClient(null);
```

Reply:
awaiting disposable objects does not work when taken out of braces.